server/uploader.js

```diff
@@ -81,84 +61,130 @@ module.exports = {
    */
   bigwigtoBcwig: function(prefix, bwFile, bigWigWoWigAddr) {
     // convert *.bw into *.wig
-    var wigFileName = bwFile.substr(0, bwFile.length - 3) + '.wig';
-    console.log("start transfer");
-    shell.exec(bigWigWoWigAddr + ' ' + prefix + bwFile + ' ' + prefix + wigFileName);
+    var wigFileName = bwFile + '.wig';
+    console.log('start transfer');
+    child.execSync(bigWigWoWigAddr + ' ' + prefix + bwFile + ' ' + prefix + wigFileName);
```

Comment:
Please avoid concatenating strings using ' '. Use Array.join. I've corrected it. Also try to reuse the same concated string.